examples/client-secure-byoc.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 The Cockroach Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+# This config file demonstrates how to connect to the CockroachDB StatefulSet
```

Comment:
This probably won't pass the linting.  Is CI passing?

Reply:
Yes.